.github/workflows/continuous-integration.yml

```diff
@@ -27,5 +27,4 @@ jobs:
     name: "PHPUnit"
     uses: "doctrine/.github/.github/workflows/continuous-integration.yml@7.1.0"
     with:
-      php-versions: '["7.1", "7.2", "7.3", "7.4", "8.0", "8.1", "8.2", "8.3"]'
-      composer-options: '--ignore-platform-req=php+'
+      php-versions: '["8.1", "8.2", "8.3"]'
```

Comment:
Why not 8.4 here? It is allowed in composer.json.

Reply:
This MR was only about dropping support for old versions.